libraries/AP_Notify/OreoLED_PX4.cpp

```diff
@@ -163,36 +163,15 @@ bool OreoLED_PX4::mode_firmware_update()
 }
 
 
-// Procedure for when Pixhawk is booting up and initializing
-// Makes all LEDs rapidly strobe blue. Returns LED_INIT_STAGE
-// 1 = Default, initialization has not yet begun
-// 2 = Initialization flag found, initialization in progress
-// 3 = Initialization flag no longer found, initialization complete
+// Makes all LEDs rapidly strobe blue while gyros initialize.
 bool OreoLED_PX4::mode_init()
 {
-    static uint16_t stage = 1;
-
-    // Pixhawk has not begun initializing yet. Strobe all blue
-    if ((!AP_Notify::flags.initialising) && ((stage == 1))) {
-        set_rgb(OREOLED_INSTANCE_ALL, OREOLED_PATTERN_STROBE, 0, 0, 255,0,0,0,PERIOD_SUPER,0);
-
-    // Pixhawk has begun initializing
-    } else if ((AP_Notify::flags.initialising) && ((stage == 1))) { 
-        stage = 2;
+    if (AP_Notify::flags.initialising){
```

Comment:
Comment 1: I think that it is better to put a whitespace between ")" and "{" in "if (AP_Notify::flags.initialising){" (Coding manners of ardupilot)
Comment 2: I think that it is better to do "return AP_Notify :: flags.initialising;" more simply.